auth-web/src/components/auth/manage-business/DashboardHelp.vue

```diff
@@ -0,0 +1,170 @@
+      </v-btn>
+    </div>
+
+    <div v-if="helpTextView" id="business-help-text" class="my-8 pt-8 pb-2">
+      <header id="help-dropdown-header" align="center" class="mb-6">
+        <strong>Help with Starting a Business</strong>
+      </header>
+      <body id="help-text-body" class="mx-12 px-12">
+        <p class="my-6">Start a named or numbered business in
+          B.C. by following these steps:
+        </p>
+        <ol>
+          <li class="py-2"><strong>Decide on a Business Type</strong></li>
+            <ul>
+              <li class="py-2">Decide which business structure is most appropriate
+                for you. A few options are: a sole proprietorship
+                partnership, or corporation. Each structure has
+                different legal and financial implications.
+                <a class="help-text-link" @click="goToSelectorWizard">
+                  Use the Business Structures Wizard to help you decide.
+                </a>
+                <v-icon small color="primary" class="mt-n1">mdi-open-in-new</v-icon>
+              </li>
+              <li class="py-2">If you want to start a corporation, you also have the
+                choice of using a named or numbered company.
+              </li>
+            </ul>
+          <li class="py-4">
+            <strong>Request a Business Name or Use a Numbered Company</strong>
+          </li>
+            <ul>
+              <li class="py-2">If you would like to start a named
+                business, the first step is is to
+                <a class="help-text-link" @click="goToNameRequest()">
+                  submit a Name Request
+                </a>.
+                Select "Request a Business Name" and create a unique name
+                for your business and submit this name for examination by
+                the Business Registry.
+              </li>
+              <li class="py-2">You do not need to submit a Name Request
+                if you are starting a numbered company.
+              </li>
+            </ul>
+          <li class="py-2"><strong>Incorporate or Register</strong></li>
+            <ul>
+              <li class="py-2">If you requested your business name
+                through your account, you will need to add your Name
+                Request to your dashboard manually using your NR
+                number. You can track the approval status of your Name
+                Request by opening it from your Business Registry dashboard.
+              </li>
+              <li class="py-2">If one of your requested names is
+                approved, you can use it to incorporate or register you
+                business by selecting "Use this Name Request Now" from
+                the dropdown menu for you Name Request.
+              </li>
+              <li class="py-2">For a numbered company, select "Start a
+                Numbered Company", and select the type of numbered
+                company you would like to start.
```

Comment:
Check with your UI/UX person: I don't think there are multiple types of numbered companies (atm anyway).